tools/templates/vfvs_run.py

```diff
@@ -267,19 +400,175 @@ def process_ligand(task):
 
     try:
         cmd = program_runstring_array(task)
+        
+        if(task['program'] == "plants"):
+            temp_file = cmd[-1]
+            os.system('cp {} {}'.format(task['config_path'], temp_file))
+            with open(temp_file, 'a+') as f: 
+                f.writelines('ligand_file {}\n'.format(task['ligand_path']))
+                f.writelines('output_dir {}\n'.format(task['output_path'])) 
+                
+        if(task['program'] == "M-Dock"):
+            os.system('cp ./mdock_dock.mol2 {}'.format(task['output_path']))
```

Comment:
where does mdock_dock.mol2 come from?

Reply:
Hmm, the output of running M-dock is suposed to be mdock_dock.mol2. Clearly, this name will clash if multiple simultaneous jobs are run. I will try to figure out how to update the mdock output file (might be just a parameter in mdock)

Reply:
We can easily have it run in a separate temp dir, I'm asking because at this point M-dock hasn't been run.  `cmd = program_runstring_array(task)` only generates the command that will be run

Reply:
Good point about the temp directory. I think this is a bug then. Will double-check.